build/release.mk

```diff
@@ -72,39 +73,66 @@ endif
 ifeq ($(BUILDER_GOOS_GOARCH),"darwin_amd64")
 	cp $(GOPATH)/bin/mattermost $(DIST_PATH)/bin # from native bin dir, not cross-compiled
 	cp $(GOPATH)/bin/platform $(DIST_PATH)/bin # from native bin dir, not cross-compiled
+
+	@echo Downloading prepackaged plugins legacy
+	@for plugin_package_legacy in $(PLUGIN_PACKAGES_LEGACY) ; do \
+		curl -s https://api.github.com/repos/mattermost/$$plugin_package_legacy | grep browser_download_url | grep darwin | cut -d '"' -f 4 | wget -qi - -P  $(DIST_PATH)/prepackaged_plugins/ ;\
+	done
 else
 	cp $(GOPATH)/bin/darwin_amd64/mattermost $(DIST_PATH)/bin # from cross-compiled bin dir
 	cp $(GOPATH)/bin/darwin_amd64/platform $(DIST_PATH)/bin # from cross-compiled bin dir
+	@echo Downloading prepackaged plugins legacy
+	@for plugin_package_legacy in $(PLUGIN_PACKAGES_LEGACY) ; do \
+		curl -s https://api.github.com/repos/mattermost/$$plugin_package_legacy | grep browser_download_url | grep darwin | cut -d '"' -f 4 | wget -qi - -P  $(DIST_PATH)/prepackaged_plugins/ ;\
+	done
 endif
 	@# Package
 	tar -C dist -czf $(DIST_PATH)-$(BUILD_TYPE_NAME)-osx-amd64.tar.gz mattermost
 	@# Cleanup
 	rm -f $(DIST_PATH)/bin/mattermost
 	rm -f $(DIST_PATH)/bin/platform
+	rm -f $(DIST_PATH)/prepackaged_plugins/*jira*
```

Comment:
What is this part for?

Reply:
to remove the platform specific, this jira release was builded for platforms so we need to remove when we package, then the next platform will get a clean dist